Tests/SwiftDocCTests/Infrastructure/AutoCapitalizationTests.swift

```diff
@@ -112,7 +112,7 @@ class AutoCapitalizationTests: XCTestCase {
         let paramsRenderSection = translatedParameters?.defaultValue?.section as! ParametersRenderSection
         
         XCTAssertEqual(paramsRenderSection.parameters.map(\.content), [
-            [SwiftDocC.RenderBlockContent.paragraph(SwiftDocC.RenderBlockContent.Paragraph(inlineContent: [SwiftDocC.RenderInlineContent.text("Upper-cased first parameter description.")]))],
+            [SwiftDocC.RenderBlockContent.paragraph(SwiftDocC.RenderBlockContent.Paragraph(inlineContent: [SwiftDocC.RenderInlineContent.text("Upper-Cased first parameter description.")]))],
```

Comment:
That doesn't look quite right, why is it "Upper-Cased" instead of "Upper-cased"

Reply:
Resolved: this is due to `localizedCapitalized`, which deems the second word after a hyphen also capitalizable. I've updated the tests so that they also call `localizedCapitalized` as well now